modules/fedimint-ln-client/src/lib.rs

```diff
@@ -214,12 +215,12 @@ impl LightningClientExt for Client {
         let (_lightning, instance) = self.get_first_module::<LightningClientModule>(&KIND);
         let mut dbtx = instance.db.begin_transaction().await;
         match dbtx.get_value(&LightningGatewayKey).await {
-            Some(active_gateway) => Ok(active_gateway),
+            Some(active_gateway) => Ok(active_gateway.info),
             None => self
                 .fetch_registered_gateways()
                 .await?
                 .into_iter()
-                .filter(|gw| !gw.is_expired())
```

Comment:
Not filtering on `is_expired` anymore?

Reply:
`fetch_registered_gateways()` now returns a list of `LightningGatewayAnnouncement`s which contain a `Duration`-based TTL, so checking for expired ones doesn't make sense since `Duration`s can't be negative.